### ISSUE TITLE
Tutorial at W3Clan pages

### ISSUE THREAD

**Issue by rebootcode** [StefanSchroeder/Golang-Regex-Tutorial#15]
Please write the tutorial at https://w3clan.com - It's easy to communicate and ask there for any help too.

**Comment by StefanSchroeder** [comment]
Hello rebootcode, thanks for your suggestion. I looked at w3clan.
I do not agree with their Terms and Conditions.

I think that the Github Issue system is good enough  to communicate and to ask for help as well.
Furthermore, with Github I have built-in version control and I cannot understand from the description on their website if they provide that on w3clan. As a matter of fact I cannot find _any_ information on how to create a tutorial. If you search for w3clan on their site, you have '0' (zero) hits. Seems, there is no tutorial on how to write a tutorial.
